src/main/java/com/example/tweet/services/TweetService.java

```diff
@@ -30,4 +30,19 @@ public int deleteTweet(int id) {
         int resultNum = tweetDao.deleteOneTweet(id);
         return resultNum;
     }
+
+    // つぶやく編集対象となるTweet1件の情報を返す
+    public Tweet selectTweetById(int tweetId) {
+        return tweetDao.selectTweetById(tweetId);
+    }
+
+    public String updateTweetById(int tweetId, String content) {
+        int updateResult = tweetDao.updateTweetById(tweetId, content);
+        // 疑似例外処理
+        if (updateResult > 0) {
+            return "tweetを更新しました。";
+        } else {
+            return "tweetの更新に失敗しました。";
+        }
```

Comment:
```suggestion
        // 疑似例外処理
        if (updateResult < 1) {
            return "tweetの更新に失敗しました。";
        } 
        
        return "tweetを更新しました。";
```
不正な場合は早期 return しておいた方が読みやすいですね。

Reply:
ここで処理結果の文字列で返しているのは、フラッシュメッセージを画面に出す構想があったとかですかね？